This one: https://github.com/sddm/sddm/actions/runs/9962123228/job/27525184808

```
-- Check for working CXX compiler: /usr/bin/clazy
CMake Error at /usr/share/cmake/Modules/CMakeTestCXXCompiler.cmake:60 (message):
  The C++ compiler
-- Check for working CXX compiler: /usr/bin/clazy - broken

    "/usr/bin/clazy"

  is not able to compile a simple test program.

  It fails with the following output:

    Change Dir: /__w/sddm/sddm/build/CMakeFiles/CMakeScratch/TryCompile-6mgS42
    
    Run Build Command(s):/usr/bin/cmake -E env VERBOSE=1 /usr/bin/gmake -f Makefile cmTC_6d0a4/fast && /usr/bin/gmake  -f CMakeFiles/cmTC_6d0a4.dir/build.make CMakeFiles/cmTC_6d0a4.dir/build
    gmake[1]: Entering directory '/__w/sddm/sddm/build/CMakeFiles/CMakeScratch/TryCompile-6mgS42'
    Building CXX object CMakeFiles/cmTC_6d0a4.dir/testCXXCompiler.cxx.o
    /usr/bin/clazy    -o CMakeFiles/cmTC_6d0a4.dir/testCXXCompiler.cxx.o -c /__w/sddm/sddm/build/CMakeFiles/CMakeScratch/TryCompile-6mgS42/testCXXCompiler.cxx
    PLEASE submit a bug report to https://github.com/llvm/llvm-project/issues/ and include the crash backtrace, preprocessed source, and associated run script.
    Stack dump:
    0.	Program arguments: clang++ -Qunused-arguments -Xclang -load -Xclang ClazyPlugin.so -Xclang -add-plugin -Xclang clazy -o CMakeFiles/cmTC_6d0a4.dir/testCXXCompiler.cxx.o -c /__w/sddm/sddm/build/CMakeFiles/CMakeScratch/TryCompile-6mgS42/testCXXCompiler.cxx
    1.	<unknown> parser at unknown location
    2.	/__w/sddm/sddm/build/CMakeFiles/CMakeScratch/TryCompile-6mgS42/testCXXCompiler.cxx:4:11: parsing function body 'main'
    3.	/__w/sddm/sddm/build/CMakeFiles/CMakeScratch/TryCompile-6mgS42/testCXXCompiler.cxx:4:11: in compound statement ('{}')
    Stack dump without symbol names (ensure you have llvm-symbolizer in your PATH or set the environment var `LLVM_SYMBOLIZER_PATH` to point to it):
    0  libLLVM.so.18.1      0x00007f9b0667acda llvm::sys::PrintStackTrace(llvm::raw_ostream&, int) + [58](https://github.com/sddm/sddm/actions/runs/9962123228/job/27525184808#step:6:59)
-- Configuring incomplete, errors occurred!
    1  libLLVM.so.18.1      0x00007f9b06678484 llvm::sys::RunSignalHandlers() + 84
    2  libLLVM.so.18.1      0x00007f9b065aa4f4
    3  libc.so.6            0x00007f9b0[59](https://github.com/sddm/sddm/actions/runs/9962123228/job/27525184808#step:6:60)896f0
    4  libclang-cpp.so.18.1 0x00007f9b0d772a88
    5  libclang-cpp.so.18.1 0x00007f9b0d79a59d clang::Preprocessor::Lex(clang::Token&) + 45
    6  libclang-cpp.so.18.1 0x00007f9b0d7bae83
    7  libclang-cpp.so.18.1 0x00007f9b0d7ad109
...
```

Apparently the latest version isn't installable, so it picks an older version:

```
2024-07-16T18:25:06.8129918Z + dnf --assumeyes --allowerasing --nobest install clang clazy
2024-07-16T18:25:07.2534779Z Last metadata expiration check: 0:00:33 ago on Tue Jul 16 18:24:34 2024.
2024-07-16T18:25:07.3517445Z Dependencies resolved.
2024-07-16T18:25:07.3640625Z 
2024-07-16T18:25:07.3642361Z  Problem: package clazy-1.11^git20240128.69fedb4-4.el9.next.x86_64 from epel-next requires clang(major) = 17, but none of the providers can be installed
2024-07-16T18:25:07.3645180Z   - package clang-17.0.1-2.el9.i686 from appstream requires clang-libs(x86-32) = 17.0.1-2.el9, but none of the providers can be installed
2024-07-16T18:25:07.3647958Z   - package clang-17.0.6-4.el9.i686 from appstream requires clang-libs(x86-32) = 17.0.6-4.el9, but none of the providers can be installed
2024-07-16T18:25:07.3650344Z   - package clang-17.0.6-5.el9.i686 from appstream requires clang-libs(x86-32) = 17.0.6-5.el9, but none of the providers can be installed
2024-07-16T18:25:07.3652557Z   - clang-libs-17.0.1-2.el9.i686 from appstream  does not belong to a distupgrade repository
2024-07-16T18:25:07.3654340Z   - clang-libs-17.0.6-4.el9.i686 from appstream  does not belong to a distupgrade repository
2024-07-16T18:25:07.3656118Z   - clang-libs-17.0.6-5.el9.i686 from appstream  does not belong to a distupgrade repository
2024-07-16T18:25:07.3658402Z   - cannot install both clang-libs-18.1.6-3.el9.x86_64 from appstream and clang-libs-17.0.1-2.el9.x86_64 from appstream
2024-07-16T18:25:07.3660568Z   - cannot install both clang-libs-18.1.6-3.el9.x86_64 from appstream and clang-libs-17.0.6-4.el9.x86_64 from appstream
2024-07-16T18:25:07.3662829Z   - cannot install both clang-libs-18.1.6-3.el9.x86_64 from appstream and clang-libs-17.0.6-5.el9.x86_64 from appstream
2024-07-16T18:25:07.3665110Z   - package clang-18.1.6-3.el9.x86_64 from appstream requires libclang-cpp.so.18.1()(64bit), but none of the providers can be installed
2024-07-16T18:25:07.3667816Z   - package clang-18.1.6-3.el9.x86_64 from appstream requires clang-libs(x86-64) = 18.1.6-3.el9, but none of the providers can be installed
2024-07-16T18:25:07.3670228Z   - cannot install both clang-libs-17.0.1-2.el9.x86_64 from appstream and clang-libs-18.1.6-3.el9.x86_64 from @System
2024-07-16T18:25:07.3672314Z   - cannot install both clang-libs-17.0.6-4.el9.x86_64 from appstream and clang-libs-18.1.6-3.el9.x86_64 from @System
2024-07-16T18:25:07.3674402Z   - cannot install both clang-libs-17.0.6-5.el9.x86_64 from appstream and clang-libs-18.1.6-3.el9.x86_64 from @System
2024-07-16T18:25:07.3676419Z   - cannot install both clang-18.1.6-3.el9.x86_64 from appstream and clang-17.0.1-2.el9.x86_64 from appstream
2024-07-16T18:25:07.3677466Z   - cannot install both clang-18.1.6-3.el9.x86_64 from appstream and clang-17.0.6-4.el9.x86_64 from appstream
2024-07-16T18:25:07.3678506Z   - cannot install both clang-18.1.6-3.el9.x86_64 from appstream and clang-17.0.6-5.el9.x86_64 from appstream
2024-07-16T18:25:07.3679885Z   - cannot install the best candidate for the job
2024-07-16T18:25:07.3680694Z ===========================================================================================
2024-07-16T18:25:07.3681631Z  Package                       Arch    Version                             Repo        Size
2024-07-16T18:25:07.3682666Z ===========================================================================================
2024-07-16T18:25:07.3683453Z Installing:
2024-07-16T18:25:07.3684026Z  clang                         x86_64  18.1.6-3.el9                        appstream   72 k
2024-07-16T18:25:07.3684946Z  clazy                         x86_64  1.11-5.el9                          epel       590 k
2024-07-16T18:25:07.3686032Z Installing dependencies:
2024-07-16T18:25:07.3687014Z  clang15                       x86_64  15.0.7-5.el9                        epel        72 k
2024-07-16T18:25:07.3688236Z  clang15-libs                  x86_64  15.0.7-5.el9                        epel        21 M
2024-07-16T18:25:07.3689348Z  clang15-resource-filesystem   x86_64  15.0.7-5.el9                        epel        11 k
2024-07-16T18:25:07.3690321Z  llvm15-libs                   x86_64  15.0.7-5.el9                        epel        23 M
2024-07-16T18:25:07.3691109Z Skipping packages with conflicts:
2024-07-16T18:25:07.3691928Z (add '--best' to command line to force their upgrade):
2024-07-16T18:25:07.3692698Z  clang                         x86_64  17.0.1-2.el9                        appstream   72 k
2024-07-16T18:25:07.3693643Z  clang                         x86_64  17.0.6-4.el9                        appstream   72 k
2024-07-16T18:25:07.3694468Z  clang                         x86_64  17.0.6-5.el9                        appstream   72 k
2024-07-16T18:25:07.3695277Z  clang-libs                    x86_64  17.0.1-2.el9                        appstream   95 M
2024-07-16T18:25:07.3696255Z  clang-libs                    x86_64  17.0.6-4.el9                        appstream   95 M
2024-07-16T18:25:07.3697081Z  clang-libs                    x86_64  17.0.6-5.el9                        appstream   23 M
2024-07-16T18:25:07.3697771Z Skipping packages with broken dependencies:
2024-07-16T18:25:07.3698617Z  clang                         i686    17.0.1-2.el9                        appstream   76 k
2024-07-16T18:25:07.3699367Z  clang                         i686    17.0.6-4.el9                        appstream   77 k
2024-07-16T18:25:07.3700372Z  clang                         i686    17.0.6-5.el9                        appstream   77 k
2024-07-16T18:25:07.3701337Z  clazy                         x86_64  1.11^git20240128.69fedb4-4.el9.next epel-next  627 k
```

--- a/.github/workflows/build.yml
+++ b/.github/workflows/build.yml
@@ -52,7 +52,7 @@ jobs:
           if [[ "${{ matrix.container-image }}" =~ "centos" ]]; then dnf --assumeyes config-manager --set-enabled crb && dnf --assumeyes install epel-release; fi
           dnf --assumeyes builddep sddm
           if [ "${{ matrix.qt }}" = "qt5" ] && [[ "${{ matrix.container-image }}" =~ "fedora" ]]; then dnf --assumeyes install qt5-qtbase-devel qt5-qtdeclarative-devel qt5-qttools-devel; fi
-          dnf --assumeyes --allowerasing --nobest install clang clazy
+          dnf --assumeyes install --allowerasing --nobest clang clazy
       - name: Dependencies (deb-type)
         if: ${{ contains(matrix.container-image, 'ubuntu') }}
         run: |

--- a/.github/workflows/build.yml
+++ b/.github/workflows/build.yml
@@ -21,7 +21,6 @@ jobs:
       matrix:
         container-image:
           - "quay.io/fedora/fedora:rawhide"
-          - "quay.io/centos/centos:stream9"
           - "registry.opensuse.org/opensuse/tumbleweed-dnf"
           - "docker.io/library/ubuntu:22.04"
           - "docker.io/library/ubuntu:23.04"
@@ -31,6 +30,9 @@ jobs:
         qt:
           - "qt5"
         include:
+          - container-image: "quay.io/centos/centos:stream9"
+            compiler: gcc
+            qt: "qt5"
           - container-image: "docker.io/library/ubuntu:23.04"
             compiler: clang
             qt: "qt6"

